packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -1,205 +1,10 @@
-import { ethers } from "ethers";
-import { Action } from "../TasitSdk";
-const { Contract, ERC20, ERC721, Marketplace } = Action;
-const { Mana } = ERC20;
-const { Estate, Land } = ERC721;
-const { Decentraland } = Marketplace;
-import DecentralandUtils from "../helpers/DecentralandUtils";
-import TasitContracts from "../../../tasit-contracts/dist";
-const { local: localContracts, ropsten: ropstenContracts } = TasitContracts;
-const { LANDProxy } = localContracts;
-const { abi: landProxyABI } = LANDProxy;
-
 // Helpers
 import actionHelpers from "tasit-action/dist/testHelpers/helpers";
 global = Object.assign(global, actionHelpers);
 
-const {
-  utils: ethersUtils,
-  constants: ethersConstants,
-  Contract: ethersContract,
-} = ethers;
-
-const setupContracts = async ownerWallet => {
-  const {
-    MANAToken,
-    LANDRegistry,
-    LANDProxy,
-    EstateRegistry,
-    Marketplace,
-  } = localContracts;
-
-  const { address: MANA_ADDRESS } = MANAToken;
-  const { address: LAND_ADDRESS } = LANDRegistry;
-  const { address: LAND_PROXY_ADDRESS } = LANDProxy;
-  const { address: ESTATE_ADDRESS } = EstateRegistry;
-  const { address: MARKETPLACE_ADDRESS } = Marketplace;
-
-  // Note: It would be cooler to use NFT here if
-  // Decentraland Land contract followed ERC721 exactly
-  const landContract = new Land(LAND_ADDRESS, ownerWallet);
-  const landProxyContract = new Contract(
-    LAND_PROXY_ADDRESS,
-    landProxyABI,
-    ownerWallet
-  );
-  const estateContract = new Estate(ESTATE_ADDRESS, ownerWallet);
-
-  const manaContract = new Mana(MANA_ADDRESS, ownerWallet);
-  const marketplaceContract = new Decentraland(
-    MARKETPLACE_ADDRESS,
-    ownerWallet
-  );
-  const landProxyContractWithLandABI = new Land(
-    LAND_PROXY_ADDRESS,
-    ownerWallet
-  );
-
-  const landProxyUpgrade = landProxyContract.upgrade(
-    LAND_ADDRESS,
-    ownerWallet.address,
-    gasParams
-  );
-  await landProxyUpgrade.waitForNonceToUpdate();
-
-  const estateInitialize = estateContract.initialize(
-    "Estate",
-    "EST",
-    landProxyContract.getAddress()
-  );
-  await estateInitialize.waitForNonceToUpdate();
-
-  const landInitialize = landProxyContractWithLandABI.initialize(
-    ownerWallet.address,
-    gasParams
-  );
-  await landInitialize.waitForNonceToUpdate();
-
-  const landEstateSetup = landProxyContractWithLandABI.setEstateRegistry(
-    estateContract.getAddress(),
-    gasParams
-  );
-  await landEstateSetup.waitForNonceToUpdate();
-
-  const marketplaceInitialize = marketplaceContract.initialize(
-    manaContract.getAddress(),
-    estateContract.getAddress(),
-    ownerWallet.address
-  );
-  await marketplaceInitialize.waitForNonceToUpdate();
-
-  return {
-    manaContract,
-    landContract: landProxyContractWithLandABI,
-    estateContract,
-    marketplaceContract,
-  };
-};
-
-const createParcels = async (landContract, parcels, beneficiary) => {
```

Comment:
Does removing this from the Tasit SDK test helpers file mean we won't be using this to help with local tests anymore?

Reply:
Yep. These functions were moved to `tasit-contract` and are run together with migrations scripts from `decentralandOrdersLoad.js`

Reply:
Ah right. Sounds good! Resolving this.